lib/literature/post.ex

```diff
@@ -33,6 +33,7 @@ defmodule Literature.Post do
     field(:feature_image_caption, :string)
     field(:featured, :boolean)
     field(:published_at, :utc_datetime)
+    field(:is_published, :boolean)
```

Comment:
@rcmonsayac why not put false as default here

Reply:
@dgross881 added